frontend/src/app/services/webhook.service.ts

```diff
@@ -13,6 +14,6 @@ export class WebhookService {
   webhookEndpoint = '/api/webhook';
 
   getWebhook(): Observable<WebHookResponse> {
-    return this.http.get<WebHookResponse>(this.webhookEndpoint);
+    return this.http.get<WebHookResponse>(this.webhookEndpoint).pipe(first());
```

Comment:
Why do we need to pipe here and deliver an Empty error?

Reply:
Piping here is done for readability. It is much easier to read this rather than `first()(this.http.get<WebHookResponse>(this.webhookEndpoint))`. Plus, if we want to add more pipeable operators, it would be much easier to modify.

Reply:
gotchu